Homework2/StackCalculator/StackCalculator.cs

```diff
@@ -0,0 +1,83 @@
+﻿namespace StackCalculator;
+
+using System;
+
+/// <summary>
+/// Class to make computing using stack.
+/// </summary>
+public class StackCalculator
+{
+    private IStack? newStack;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="StackCalculator"/> class.
+    /// </summary>
+    /// <param name="stack">stack element for StackCalculator.</param>
+    public StackCalculator(IStack stack)
+       => this.newStack = stack;
+
+    /// <summary>
+    /// Method calculate expression in postfix form and return result.
+    /// </summary>
+    /// <param name="commandSequence">expression in postfix form.</param>
+    /// <returns>result of computing. Also return true if method completed the work correctly. Else false.</returns>
+    public (float Result, bool CheckOfCorrectWork) CalculateInPostfixForm(string commandSequence)
+    {
+        var arrayOfNumbersAndOperators = commandSequence.Split();
+        var checkOfCorrectWork = true;
+        for (int i = 0; i < arrayOfNumbersAndOperators.Length; ++i)
+        {
+            if (arrayOfNumbersAndOperators[i] == "*" || arrayOfNumbersAndOperators[i] == "/"
+            || arrayOfNumbersAndOperators[i] == "+" || arrayOfNumbersAndOperators[i] == "-")
```

Comment:
```suggestion
                    || arrayOfNumbersAndOperators[i] == "+" || arrayOfNumbersAndOperators[i] == "-")
```
Это перенос предыдущей строки, обычно выделяется двумя отступами